experimental/components/ConversationalCore/assistant.dialog

```diff
@@ -0,0 +1,83 @@
+{
+  "$kind": "Microsoft.AdaptiveDialog",
+  "$designer": {
+    "$designer": {
+      "name": "Assistant",
```

Comment:
What does it mean here that the name of this "bot" is Assistant? When the ConversationalCore is imported, do users see Assistant at all?

Reply:
So, realistically, this is something that should be templatized. I think this something we'll need to figure out in conjunction with the generators and overall templating strategy. I agree that it shouldn't be 'Assistant' out of the box - but until we figure out the templating stuff, I'm not sure that I can do much right now. I'll create an issue to track this so we can make sure it gets resolved.

Reply:
Opened https://github.com/microsoft/botframework-components/issues/451.